pkg/apis/v1/managedzone_types.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2022 The MultiCluster Traffic Controller Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ManagedZoneReference holds a reference to a ManagedZone
+type ManagedZoneReference struct {
+	// `namespace` is the namespace of the managed zone
+	// Required
+	Namespace string `json:"namespace"`
```

Comment:
Wondering if this is needed at this point? or would same namespace suffice for now?

Reply:
I think we can probably assume everything is in the same ns as the MZ now. If that requirement changes we can deal with it later.